packages/core/tests/dummy/app/routes/table.ts

```diff
@@ -56,7 +56,7 @@ export default class TableRoute extends Route {
     let rows = A();
 
     //20k rows
```

Comment:
Is this number still the case?

Reply:
Yes it was originally 4000, but I lowered it for testing
https://github.com/yahoo/navi/blob/bb557d4fe9c4acfe9908a73c97842368cbb850a7/packages/core/tests/dummy/app/routes/table.js#L54-L55